encoder/irc_datatypes.h

```diff
@@ -37,7 +37,7 @@ typedef unsigned long long   UWORD64;
 typedef signed char     WORD8;
 typedef short           WORD16;
 typedef int             WORD32;
-
+typedef int64_t WORD64;
```

Comment:
please maintain the indentation like previous line (i agree that clang-format would change it, but still for this case let us keep it consistent with previous line)